src/lib/Time.js

```diff
@@ -0,0 +1,46 @@
+const timeUtils = require('../utils/time_utils.js');
+
+let simTimeSub = null;
+let simTime = timeUtils.dateToRosTime(0);
+
+function handleSimTimeMessage(msg) {
+  console.log('Got sim time update: %j', msg);
+  simTime = msg.clock;
+}
+
+const Time = {
```

Comment:
We should create an Issue to track the fact that Sim time will not be represented in any of the logs (they will use the wall-clock time).